xunit-autolabeler-v2/ast_parser/python/source_parser_test.py

```diff
@@ -0,0 +1,236 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+import unittest
+from unittest.mock import MagicMock
+
+
+from . import source_parser
+
+import pytest
+
+
+class GetMethodChildrenTest(unittest.TestCase):
```

Comment:
This Test looks pretty repetitive, consider using parameterized:
https://pypi.org/project/parameterized/

Reply:
I cleaned this up a bit, but I'm not sure it's repetitive _enough_ to use `parametrize`.

(There are some differences between the tests that IMO would be hard to represent that way.)